### PR TITLE
docs: Add README

### LINKED ISSUE #1: Purpose of this repo
@gnawux What is this repo for? Compared to the community and the website one, what does it add?

### PR DESCRIPTION
**Pull request by jodh-intel**
Add a `README.md` that explains the purpose of this repository along
with pointers to the other repositories.

Fixes #1.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
Hi @sameo, @gnawux - WDYT?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+##### Agent
+
+The [`kata-agent`](https://github.com/kata-containers/agent) runs inside the
+virtual machine and sets up the container environment.
+
+##### KSM throttler
+
+The [`kata-ksm-throttler`](https://github.com/kata-containers/ksm-throttler)
+is an optional utility that monitors containers and compacts memory to
```
Comment (sameo) [inline README.md]: deduplicates memory

Comment (jodh-intel) [reply README.md]: Good catch - fixed.

**Comment by sameo** [comment]
LGTM

**Pull request merged**